test/functional/mempool_limit.py

```diff
@@ -85,6 +85,99 @@ def test_rbf_carveout_disallowed(self):
         assert_equal(res["package_msg"], "transaction failed")
         assert "too-long-mempool-chain" in res["tx-results"][tx_C["wtxid"]]["error"]
 
+    def test_mid_package_eviction_success(self):
+        node = self.nodes[0]
+        self.log.info("Check a package where each parent passes the current mempoolminfee but a parent could be evicted before getting child's descendant feerate")
+
+        # Clear mempool so it can be filled with minrelay txns
+        self.restart_node(0, extra_args=self.extra_args[0] + ["-persistmempool=0"])
+        assert_equal(node.getrawmempool(), [])
+
+        # Restarting the node resets mempool minimum feerate
```

Comment:
nit: relying on the fact that mempool min feerate isn't persisted seems a little bit brittle.

Reply:
If someone opens a PR to persist I'll push for an RPC to set it explicitly :)